scripts/globals/caskets.lua

```diff
@@ -401,10 +402,23 @@ function getDrops(npc, dropType, zoneId)
         local tempCount    = 1
         local randomTable  = {1,3,1,2,1,2,1,1,3,1,2,1}
 
+        if zoneId == 172 or zoneId == 173 or zoneId == 190 or
+           zoneId == 191 or zoneId == 193 or zoneId == 194 or
+           zoneId == 196 or zoneId == 198 then
```

Comment:
Only thing I think might be improved on is defining these zones as a Set in the caskets global:
```lua
casketInfo.splitZones = set{
    tpz.zone.ZERUHN_MINES,
    tpz.zone.KORROLOKA_TUNNEL
}
```

And then your check here becomes:
`if casketInfo.splitZones[zoneId] then`

Reply:
yep sounds good